store/src/store.rs

```diff
@@ -83,6 +92,18 @@ pub trait ChainStore<'a>: Send + Sync + Sized {
         .collect()
     }
 
+    fn get_unfrozen_block(&'a self, h: &packed::Byte32) -> Option<BlockView> {
```

Comment:
Maybe need to write a panic warning here? Or turn any panic into None and return?

Reply:
This copy from 
```
    fn get_block(&'a self, h: &packed::Byte32) -> Option<BlockView> {
        self.get_block_header(h).map(|header| {
            let body = self.get_block_body(h);
            let uncles = self
                .get_block_uncles(h)
                .expect("block uncles must be stored");
            let proposals = self
                .get_block_proposal_txs_ids(h)
                .expect("block proposal_ids must be stored");
            BlockView::new_unchecked(header, uncles, body, proposals)
        })
```
It fail-fast on inconsistency when header exists but uncles or proposals not.

Reply:
I'm not going to change this in this PR